src/tests/Microsoft.PowerFx.Connectors.Tests/PowerPlatformConnectorTests.cs

```diff
@@ -75,6 +78,68 @@ public async Task Connector1()
             Assert.Equal(expected, actual);
         }
 
+        [Fact]
+        public async Task AzureBlobConnector_UploadFile()
+        {
+            var testConnector = new LoggingTestServer(@"Swagger\AzureBlobStorage.json");
+            var apiDoc = testConnector._apiDocument;                       
+            var config = new PowerFxConfig();
+            var token = @"AuthToken2";
+
+            HttpClient client = new PowerPlatformConnectorClient(
+                "firstrelease-001.azure-apim.net",      // endpoint
+                "839eace6-59ab-4243-97ec-a5b8fcc104e4", // environment
+                "453f61fa88434d42addb987063b1d7d2",     // connectionId
+                () => $"{token}",
+                new HttpClient(testConnector))
+            {
+                SessionId = "ccccbff3-9d2c-44b2-bee6-cf24aab10b7e"
+            };
+
+            var funcs = config.AddService("AzureBlobStorage", apiDoc, client);
+
+            // Function we added where specified in MSNWeather.json
+            var funcNames = funcs.Select(func => func.Name).OrderBy(x => x).ToArray();
+            Assert.Equal(funcNames, new string[] { "AppendFile", "CopyFile", "CopyFile_Old", "CreateFile", "CreateFile_Old", "DeleteFile", "DeleteFile_Old", "ExtractFolder_Old", "ExtractFolderV2", "GetDataSetsMetadata", "GetFileContent", "GetFileContent_Old", "GetFileContentByPath", "GetFileContentByPath_Old", "GetFileMetadata", "GetFileMetadata_Old", "GetFileMetadataByPath", "GetFileMetadataByPath_Old", "ListAllRootFolders", "ListAllRootFoldersV2", "ListFolder", "ListFolder_Old", "ListFolderV2", "ListRootFolder", "ListRootFolder_Old", "ListRootFolderV2", "TestConnection", "UpdateFile", "UpdateFile_Old" });
+
+            // Now execute it...
+            var engine = new RecalcEngine(config);            
+            testConnector.SetResponseFromFile(@"Responses\AzureBlobStorage_Response.json");
+
+            var result = await engine.EvalAsync(
+                @"AzureBlobStorage.CreateFile(""container"", ""bora1.jpg"", ""abc"")",
```

Comment:
>jpg [](http://example.com/codeflow?start=69&length=3)

for test sanity ... 
should this be a .txt extension?

Reply:
sure